source/accessories/uinverter/hardware-design.md

```diff
@@ -185,6 +185,21 @@ V_\text{amp} &= (\text{amplifier gain}) \times (\text{current} \times R_\text{se
 When the DC supply voltage exceeds 12V, the phase current increases significantly and may push the current sense amplifier in saturation.
 To aviod any damage to components on the AMDC board, the DC supply voltage must be limited to no more than 12V.  
 
+### DC link voltage sensing filter
+
+A resistive potential divider ($R_1$ and $R_2$) is provided for DC bus voltage sensing. 
+To filter out high frequency noise in DC link voltage measurement, an additional capacitor can be placed at the voltage sensing point to form a low pass filter (see image). This filter should not affect the nominal sampling operation of the DC bus voltage, i.e. we should still be able to capture actual variations in DC link voltage.
+
+![DC link voltage sensing filter](images/uInverter_DC_link_filter.svg)
+
+An important thing to consider while placing this capacitor is its sizing. Bandwidth of the low pass filter in this particular configuration is given by:
```

Comment:
Good, but please also wordsmith this description to be make this direct and succinct.